config/deploy/docker/lib/roll_out.rb

```diff
@@ -522,7 +522,7 @@ def _poll_until_deploy_tasks_complete!
         puts "[WARN] We expected: #{self.deployment_id}"
         puts "[WARN] We got: #{response.dig('registered_deployment_id')}"
         puts "[WARN] You can safely (p)roceed if this is the first deployment"
-        print "\nYou can (w)ait, (p)roceed with deployment anyway, or (a)bort: "
+        print "\nYou can (w)ait, (p)roceed with deployment anyway, (v)iew log tail, or (a)bort: "
```

Comment:
'v' seems to show the whole log vs `_tail_logs`?
Seems like a `case` might be better here?